app/views/CYMinusFourYesNoView.scala.html

```diff
@@ -0,0 +1,54 @@
+@*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import controllers.routes._
+@import config.FrontendAppConfig
+
+@this(
+        main_template: MainTemplate,
+        formHelper: FormWithCSRF,
+        appConfig: FrontendAppConfig
+)
+
+@(form: Form[_], taxYear: String, mode: Mode)(implicit request: Request[_], messages: Messages)
+
+@main_template(
+    title = s"${errorPrefix(form)} ${messages("cyMinusFour.liability.title", taxYear)}"
+) {
+
+    @formHelper(action = CYMinusFourLiabilityController.onSubmit(mode), 'autoComplete -> "off") {
+
+        @components.back_link()
+
+        @components.error_summary(form.errors)
+
+        <h1 class="heading-large">@messages("cyMinusFour.liability.heading", taxYear)</h1>
```

Comment:
Does not follow the design pattern for radio options. i.e. H1 should be within legend: 
https://design-system.service.gov.uk/components/radios/
Applies for all YesNo views in PR.

Reply:
When there's content between the H1 and radio buttons it should just have the H1 text repeated in the fieldset and hidden. Adam has confirmed this with DAC